src/code/audio_seqplayer.c

```diff
@@ -507,17 +536,20 @@ void AudioSeq_SeqLayerProcessScript(SequenceLayer* layer) {
     }
 
     AudioSeq_SeqLayerProcessScriptStep1(layer);
-    val = AudioSeq_SeqLayerProcessScriptStep2(layer);
-    if (val == -1) {
+
+    cmd = AudioSeq_SeqLayerProcessScriptStep2(layer);
+    if (cmd == -1) {
         return;
     }
 
-    val = AudioSeq_SeqLayerProcessScriptStep3(layer, val);
-    if (val != -1) {
-        val = AudioSeq_SeqLayerProcessScriptStep4(layer, val);
+    cmd = AudioSeq_SeqLayerProcessScriptStep3(layer, cmd);
+
+    if (cmd != -1) {
+        cmd = AudioSeq_SeqLayerProcessScriptStep4(layer, cmd);
     }
-    if (val != -1) {
-        AudioSeq_SeqLayerProcessScriptStep5(layer, val);
+
+    if (cmd != -1) {
+        AudioSeq_SeqLayerProcessScriptStep5(layer, cmd);
```

Comment:
Oh, that is weird that `AudioSeq_SeqLayerProcessScriptStep4` returns `sameSound` (`sameTunedSample`) instead of a command. Yeah, defs worth a comment.

I wonder if that define should be for the "layer" interpreter, or kept to a more "general" interpreter for "sequences" and "channels" two, since there are basically 3 different languages in here. I'll look into it. Either way, the name should reflect which of the scripts it's for